### PR TITLE
add web service to fastly setup

### PR DESCRIPTION
**Pull request by evgeni**
- **allow setting domains manually**
- **allow configuring healthcheck path**
- **add web service to fastly setup**

### CODE REVIEW

File: ansible/fastly.yml [hunk 1]
```diff
@@ -10,8 +10,7 @@
       fastly_service:
         name: "{{ item.service }}.theforeman.org"
         fastly_api_key: "{{ fastly_api_key }}"
-        domains:
-          - name: "{{ item.service }}.theforeman.org"
+        domains: "{{ item.domains | default([{'name': item.service ~ '.theforeman.org'}]) }}"
```
Comment (ekohl) [inline ansible/fastly.yml:13]: What's `~` here?

Comment (evgeni) [reply ansible/fastly.yml:13]: String concat.

Comment (evgeni) [reply ansible/fastly.yml:13]: https://jinja.palletsprojects.com/en/stable/templates/#other-operators

Comment (ekohl) [reply ansible/fastly.yml:13]: I guess that and tried to look for that but somehow I couldn't find it.

**Review by ekohl — APPROVED** [review]

File: ansible/fastly.yml (hunk 1 above)
Comment (evgeni) [reply ansible/fastly.yml:13]: documenting things properly is complicated, as you know :)

**Pull request merged**